packages/esm-appointments-app/translations/en.json

```diff
@@ -1,4 +1,5 @@
 {
+  "action": "Action",
```

Comment:
These 15 or so translations have already been translated into several languages in Patient Chart.  @ibacher  not sure if there's an easy way to automate bringing these over, but, in lieu of that, since there only 15 or so I'm fine with manually adding the translations to transifex from the existing translations.